lib/App/DuckPAN.pm

```diff
@@ -207,6 +207,12 @@ sub get_reply {
 	return $return;
 }
 
+sub ask_yn {
```

Comment:
I feel that using `Term::UI::ask_yn` is useful since it does a couple of things automatically to save some code:
1. It adds a '[y/n]' at the end of the prompt with the default option capitalized and
2. Returns a boolean value after parsing the reply

But I understand your concern about defining a sub that is used only once, so I'll go with anything you're comfortable having in the code base.